### ISSUE TITLE
No newline after spinners

### ISSUE THREAD

**Issue by bkabrda** [verigak/progress#12]
Reproducer:

``` python
from progress import spinner, bar

s = spinner.Spinner('Foo ... ')
s.next()
s.finish()
```

There is no newline after this spinner, so my terminal prompt is on the same line, e.g.:

```
Foo ... \[bkabrda@zizalka test]$
```

I guess the problem is in WriteMixin (used by spinners), which doesn't print newline, as opposed to WritelnMixin (used by bars) which prints new line.

**Comment by zkqiang** [comment]
The simple solution
```
s = spinner.Spinner('Foo ... ')
s.next()
if s.index > 0:
    s.write('\n')
s.finish()
```

**Comment by verigak** [comment]
This was fixed on d9d4073
